packages/http/src/mocker/errors.ts

```diff
@@ -0,0 +1,8 @@
+import { Omit } from '@stoplight/types';
+import { ProblemJson } from '../types';
+
+export const NO_INVALID_RESPONSE_TEMPLATE: Omit<ProblemJson, 'detail'> = {
+  name: 'NO_INVALID_RESPONSE_TEMPLATE',
+  title: 'Missing invalid response or response example for the current invalid request',
```

Comment:
This is a bit of a mouthful! What does "Missing invalid response" mean and what does the other half mean? 

Maybe we can just call this INVALID_REQUEST we do not need to talk about the response template being missing, thats an implementation choice we currently make which might change. The error is just that thr request sucks, and we can talk about response definitions in the `details` perhaps.

Reply:
This error means:

1. The user sends a request to Prism
2. Prism's been able to find a matching path and method
3. Prism has validated the provided payload against the JSON Schema defined in the document
4. The validation has *failed*
5. Prism looks for a 4xx response to return to the user as an error
6. No such error is found and Prism complains with this error

Does this clarify what's going on? I'm open to better title and messages, I admin they're not very helpful.

Reply:
Sure I understand the flow that lead to the error, but I think we are being to specific about why we decided to throw that. 

The error is really "validation failed" and the information should be about that. `details` can mention _why_ they are seeing this error and not their own validation messages, and that will also be in the logs, but I think we are being too specific.

Imagine if we decide to change this, so prism does not look for a user defined 400 by default, or is moved to a config switch. It would mean our overly specific error messages are now inaccurate.

Reply:
f36ccb2